bin/level/app.rs

```diff
@@ -1,14 +1,17 @@
 use cgmath;
 use gfx;
 
-use boilerplate::{Application, KeyboardInput};
+use boilerplate::{Application, KeyboardInput, MouseScrollDelta, MouseButton, ElementState};
 use vangers::{config, level, render, space};
 
 #[derive(Debug)]
 enum Input {
     Hor { dir: f32, alt: bool },
     Ver { dir: f32, alt: bool },
     Dep { dir: f32, alt: bool },
+    DepQuant {dir: f32},
```

Comment:
could just do `DepQuant(f32)` and same for the other of added variants, assuming you aren't going to add more fields any time soon
Note: I used the named `dir` and `alt` mostly because storing pure `bool` without a name is always ambiguous